lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -190,23 +190,23 @@ def patch_cache_store_read
       cache_store_class(:read).class_eval do
         alias_method :read_without_datadog, :read
         def read(*args, &block)
-          raw_payload = {
+          payload = {
             action: 'GET',
             key: args[0],
             tracing_context: {}
           }
 
           begin
             # process and catch cache exceptions
-            Datadog::Contrib::Rails::ActiveSupport.start_trace_cache(raw_payload)
+            Datadog::Contrib::Rails::ActiveSupport.start_trace_cache(payload)
             read_without_datadog(*args, &block)
           rescue Exception => e
             payload[:exception] = [e.class.name, e.message]
```

Comment:
here it was called `payload`. Changed everything to `payload` for consistency with other instrumentation.